examples/bpi-monolith-test-suite/bpi/components/zkmock/agreementalt.ts

```diff
@@ -0,0 +1,130 @@
+import { OrderAlt } from "./orderalt";
```

Comment:
again similar statement about abstraction for domain objects and how they are stored and then combined into a final agreement "worldstate" so to speak